Source/Parallelization/WarpXComm_K.H

```diff
@@ -522,6 +522,13 @@ void warpx_interp (const int j,
 {
     using namespace amrex;
 
+    // Pad input array with zeros beyond ghost cells
+    // for out-of-bound accesses due to large-stencil operations
+    const auto arr_safe = [arr_fine] (const int jj, const int kk, const int ll) noexcept
```

Comment:
I think this could even be a `constexpr` function
```suggestion
    constexpr auto arr_safe = [arr_fine] (const int jj, const int kk, const int ll) noexcept
```

Reply:
Thanks for the suggestions, Axel!

About the variable names, since this is a device kernel, I thought it would be a good idea to use a naming convention that is often used in AMReX kernels, with the tags `src` and `dst` to denote source and destination quantities, respectively. Hence, I renamed the variables as follows:
- `arr_aux` renamed as `dst_arr`
- `arr_fine` renamed as `src_arr`
- `arr_stag` renamed as `src_stag` (since it is the index type of the source MultiFab, the old name was not clear)
- `arr_safe` renamed as `src_arr_zeropad` (with the `zeropad` tag as you suggested)

About the `constexpr`, I could not use it because the compiler says that the type of the variable is not literal, "is a closure type, which is only literal in C++17 and later".